inst/tests/tests.Rraw

```diff
@@ -16066,6 +16066,22 @@ test(2111.03, Encoding(names(tbl)), c('latin1', 'latin1'))
 setnames(tbl, latin1, utf8)
 test(2111.04, Encoding(names(tbl)), c('UTF-8', 'UTF-8'))
 
+# fwrite scipen, #2020
+oldSciPen = options('scipen')
+testNo = 2112
+dec = 0L
+for (i in -3:3) {
+  for (j in -3:3) {
+    options(scipen=j)  # influence the value of base::format() being tested against
+    # use format to assure the output is the same as in R
+    v = (-1)^(i+j)
```

Comment:
I'm not sure what this test is testing.  I added `cat(i,j,format(v),"\n")` locally and this is the output.
```
-3 -3 1 
-3 -2 -1 
-3 -1 1 
-3 0 -1 
-3 1 1 
-3 2 -1 
-3 3 1 
-2 -3 -1 
-2 -2 1 
-2 -1 -1 
-2 0 1 
-2 1 -1 
-2 2 1 
-2 3 -1 
-1 -3 1 
-1 -2 -1 
-1 -1 1 
-1 0 -1 
-1 1 1 
-1 2 -1 
-1 3 1 
0 -3 -1 
0 -2 1 
0 -1 -1 
0 0 1 
0 1 -1 
0 2 1 
0 3 -1 
1 -3 1 
1 -2 -1 
1 -1 1 
1 0 -1 
1 1 1 
1 2 -1 
1 3 1 
2 -3 -1 
2 -2 1 
2 -1 -1 
2 0 1 
2 1 -1 
2 2 1 
2 3 -1 
3 -3 1 
3 -2 -1 
3 -1 1 
3 0 -1 
3 1 1 
3 2 -1 
3 3 1 
```

Reply:
I think I meant to multiply `v` by `10^j`...